plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/submitit_launcher.py

```diff
@@ -143,6 +143,13 @@ def launch(
             )
 
         jobs = executor.map_array(self, *zip(*job_params))
+        if self.config.hydra.launcher.submitit_nonblocking:
+            null_returns = []
+            for _ in jobs:
+                null_returns.append(JobReturn())
+                null_returns[-1].status = JobStatus.COMPLETED
```

Comment:
I think `JobStatus.UNKNOWN` would technically be a more accurate description than `JobStatus.COMPLETED`.